### PR TITLE
chore(springboot): replace deprecated api version in cronjob template

### PR DESCRIPTION
**Pull request by Zilv1nas**

### CODE REVIEW

File: charts/springboot/templates/cronjobs.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{ range $name, $definition := .Values.cronJobs }}
 ---
-apiVersion: "batch/v1beta1"
+apiVersion: "batch/v1"
```
Comment (KarolisL) [inline charts/springboot/templates/cronjobs.yaml:3]: Consider using (Capabilities)[https://helm.sh/docs/chart_template_guide/builtin_objects/]:

```apiVersion: "{{- if Capabilities.APIVersions.Has "batch/v1" }} "batch/v1" {{- else -}} "batch/v1beta1" {{- end }}"```

This way you'd support old version as well.

N.B. untested code, check whitespace and adjust `-` inside `{{ }}` accordingly.

Comment (Zilv1nas) [reply charts/springboot/templates/cronjobs.yaml]: Good idea 👍, updated

Comment (modax) [reply charts/springboot/templates/cronjobs.yaml]: LGTM

**Review by modax — COMMENTED** [review]
LGTM

**Review by modax — APPROVED** [review]

**Pull request merged**